### PR TITLE
Manage SVG images

### PR DESCRIPTION
**Pull request by asaunier**
Related to https://github.com/c2corg/v6_ui/issues/944

### CODE REVIEW

File: c2corg_ui/templates/image/helpers/view.html [hunk 1]
```diff
@@ -1,10 +1,13 @@
-<%def name="create_image_link(url, suffix, title)">\
+<%def name="create_image_link(image, locale, size)">\
   <%
-    i = url.rfind('.')
-    imageUrl = url[:i] + suffix + url[i:]
+    original_url = image_url + image['filename']
+    i = original_url.rfind('.')
+    resized_url = original_url[:i] + size + original_url[i:]
+    if image.get('has_svg'):
+        original_url = original_url[:i] + '.svg'
```
Comment (asaunier) [inline c2corg_ui/templates/image/helpers/view.html]: See my comments at https://github.com/c2corg/v6_images/pull/26#issuecomment-263311086

**Comment by asaunier** [comment]
I have updated the PR in order to use approach 1 described at https://github.com/c2corg/v6_images/pull/26#issuecomment-263311086

(not tested yet, the migration script fails because of username duplicate and I'm having troubles recreating my DB)

**Comment by asaunier** [comment]
I have rebased all the recent fixup commits. Please review.

**Comment by asaunier** [comment]
No one?
The API and image backends PR have been merged. Which means problems will most likely occur with SVG image unless this PR is merged.

**Comment by arnaud-morvan** [comment]
Not sure about everything, but it seems good to me.

**Comment by arnaud-morvan** [comment]
I need this to update the demov6.

**Comment by asaunier** [comment]
Merged!

**Pull request merged**